scripts/youtube_retrieve.py

```diff
@@ -0,0 +1,91 @@
+"""
+Retrieve XML dumps from specified video IDs for debugging
+
+
+"""
+import django
+import fnmatch
+import os
+from os.path import expanduser
+import pysftp
+import shutil
+import sys
+
+project_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+if project_path not in sys.path:
+    sys.path.append(project_path)
+os.environ['DJANGO_SETTINGS_MODULE'] = 'VEDA.settings'
+django.setup()
```

Comment:
@yro  Why do we have to do this in every file, it's not an good thing from Django perspective. Once we understand it's purpose we're planning to remove this from individual files.

Reply:
@muzaffaryousaf This is just a one-off script, but it needs to find the django project in order to interact with the ORM. This doesn't run as part of the platform, rather, as a sort of management command.

Reply:
@muzaffaryousaf I'll just delete this file entirely.